src/node.ts

```diff
@@ -2,19 +2,20 @@
  * Stoplight node types
  */
 export enum NodeType {
-  Article = 'article',
-  HttpService = 'http_service',
-  HttpServer = 'http_server',
-  HttpOperation = 'http_operation',
-  Model = 'model',
-  Unknown = 'unknown',
+  Article = "article",
+  HttpService = "http_service",
+  HttpServer = "http_server",
+  HttpOperation = "http_operation",
+  Model = "model",
+  Raw = "raw",
```

Comment:
https://github.com/stoplightio/platform-internal/pull/3225#pullrequestreview-436583900
Might be worth looking at it before approving. :D